docs/en/observability/grant-cases-access.asciidoc

```diff
@@ -37,6 +37,11 @@ NOTE: These privileges also enable you to delete comments and alerts from a case
 |=== 
 // lint enable observability
 
+NOTE: If you are using an on-premises {kib} deployment and you want the email
+notifications and the external incident management systems to contain links back
+to {kib}, you must configure the 
+{kibana-ref}/settings.html#server-publicBaseUrl[server.publicBaseUrl] setting.
```

Comment:
```suggestion
NOTE: If you are using an on-premises {kib} deployment and want your email
notifications and external incident management system to contain links back
to {kib}, configure the 
{kibana-ref}/settings.html#server-publicBaseUrl[server.publicBaseUrl] setting.
```
Would there be more than one external incident management system in this scenario? If no, I would make this singular. I'm also trying to make this sentence a bit shorter for easier parsing. My post-vacay brain is not sure this is better, but it is shorter. :-P

Reply:
Each case can use a different connector to a different external incident management system, but to have backlinks in *any* of them, this setting needs to exist.  So I think the plural is correct.  I'll incorporate the rest of your suggestion, however!